.github/workflows/changelog_generation.yml

```diff
@@ -24,14 +23,10 @@ jobs:
           pip install pyyaml bs4
       - name: Compile CL
         run: python tools/changelog/ss13_genchangelog.py html/changelog.html html/changelogs
-      - name: Commit
+      - name: Commit And Push
         run: |
           git config --local user.email "NebulaSS13Bot@gmail.com"
           git config --local user.name "NebulaSS13Bot"
           git pull origin dev
```

Comment:
Base on the logs, the `Checkout`-step above has already cloned/fetched/checked out dev for us, so this is another unnecessary line